pkg/webservice/dao/application_info.go

```diff
@@ -44,6 +48,14 @@ type ApplicationDAOInfo struct {
 	MaxRequestPriority int32                   `json:"maxRequestPriority,omitempty"`
 }
 
+type ApplicationDetailsDAOInfo struct {
```

Comment:
Apologies for the late review, I was on vacation for 2 weeks.

I'd rather put this in `ApplicationDAOInfo`. In fact, retrieving the summary is a fast operation, so I'm not sure that we need `details=true/false` in the URL. Unless there's an objection from someone, I'd just add these new fields to the existing `ApplicationDAOInfo`

Reply:
done!